processes/decision_process.rst

```diff
@@ -153,18 +153,18 @@ These might be:
 Creating an RFD
 ---------------
 
-Create an RFD by making a JIRA issue in the `Request For Comments (RFC) <https://jira.lsstcorp.org/projects/RFC>`_ project with a **component of DM** and a **issue type of RFD**.
-Use the **Location** field to specify the date and time of the discussion (:ref:`see below for time slot <decision-making-rfd-time>`).
+Create an RFD by making a post on community in the  `DM Request for Discussion category <https://community.lsst.org/c/dm/dm-rfd>`_.
```

Comment:
I've called the category DM RFD to fit in the column.

In addition, you can include this short link to creating a new topic:  https://community.lsst.org/new-topic?category=data%20management/dm%20rfd